src/views/Pools/components/PoolsTable/ActionPanel/ActionPanel.tsx

```diff
@@ -0,0 +1,241 @@
+import React from 'react'
+import styled, { keyframes, css } from 'styled-components'
+import {
+  Box,
+  Button,
+  Flex,
+  HelpIcon,
+  Link,
+  LinkExternal,
+  MetamaskIcon,
+  Skeleton,
+  Text,
+  TimerIcon,
+  useTooltip,
+} from '@pancakeswap/uikit'
+import { BASE_BSC_SCAN_URL, BASE_URL } from 'config'
+import { useBlock, useCakeVault } from 'state/hooks'
+import BigNumber from 'bignumber.js'
+import { Pool } from 'state/types'
+import { useTranslation } from 'contexts/Localization'
+import Balance from 'components/Balance'
+import { CompoundingPoolTag, ManualPoolTag } from 'components/Tags'
+import { getAddress } from 'utils/addressHelpers'
+import { registerToken } from 'utils/wallet'
+import { getBalanceNumber } from 'utils/formatBalance'
+import Harvest from './Harvest'
+import Stake from './Stake'
+import Apr from '../Apr'
+
+const expandAnimation = keyframes`
+  from {
+    max-height: 0px;
+  }
+  to {
+    max-height: 500px;
+  }
+`
+
+const collapseAnimation = keyframes`
+  from {
+    max-height: 500px;
+  }
+  to {
+    max-height: 0px;
+  }
+`
+
+const StyledActionPanel = styled.div<{ expanded: boolean }>`
+  animation: ${({ expanded }) =>
+    expanded
+      ? css`
+          ${expandAnimation} 300ms linear forwards
+        `
+      : css`
+          ${collapseAnimation} 300ms linear forwards
+        `};
+  overflow: hidden;
+  background: ${({ theme }) => theme.colors.dropdown};
+  display: flex;
+  flex-direction: column-reverse;
+  justify-content: center;
+  padding: 24px;
+
+  ${({ theme }) => theme.mediaQueries.lg} {
+    flex-direction: row;
+    padding: 16px 32px;
+  }
+`
+
+const ActionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    flex-direction: row;
+    align-items: center;
+    flex-grow: 1;
+    flex-basis: 0;
+  }
+`
+
+type MediaBreakpoints = {
+  isXs: boolean
+  isSm: boolean
+  isMd: boolean
+  isLg: boolean
+  isXl: boolean
+}
```

Comment:
This is not exported from the toolkit? If not, then can you add a TODO here to do so.